File: app/src/testEndToEnd/java/org/example/product/app/endtoend/SystemQuickTest.java
package org.example.product.app.endtoend;

import static org.junit.jupiter.api.Assertions.assertEquals;

import com.google.common.collect.ImmutableList;
import org.example.product.app.mock.api.MockServer;
import org.junit.jupiter.api.Test;

public class SystemQuickTest {

    @Test
    void testSome() {
        assertEquals(0, sendStuff(new MockServer().get()));
    }

    private int sendStuff(ImmutableList<String> things) {
        return 0;
    }
}

File: app/src/testEndToEnd/kotlin/org/example/product/app/endtoend/SystemQuickTest.kt
package org.example.product.app.endtoend

import com.google.common.collect.ImmutableList
import org.example.product.app.mock.api.MockServer
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class SystemQuickTest {
    @Test
    fun testSome() {
        Assertions.assertEquals(0, sendStuff(MockServer().get()))
    }

    private fun sendStuff(things: ImmutableList<String>): Int {
        return 0
    }
}
